### PR TITLE
v1.14 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.14.0 Release
+- Release Date: November 16, 2018
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported
+
+**Combatibility Note: Mobile App v1.13+ is required for Mattermost Server v5.4+**
+
+### Bug Fixes
+- Fixed an issue where the Android app did not allow establishing a network connection with any server that used a self-signed certificate that had the CA certificate installed on the device.
```
Comment (bbodenmiller) [inline CHANGELOG.md]:
```suggestion
- Fixed an issue where the Android app did not allow establishing a network connection with any server that used a self-signed certificate that had the CA certificate user installed on the device.
```

Comment (bbodenmiller) [reply CHANGELOG.md]: Not perfect but I think you need to emphasize that the issue was with user installed CAs not being used.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.14.0 Release
+- Release Date: November 16, 2018
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported
```
Comment (bbodenmiller) [inline CHANGELOG.md]: `Self-Signed SSL Certificates are not supported`

I think is misleading. They are supported if the user installs the CA cert on their device.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
 
+## 1.14.0 Release
+- Release Date: November 16, 2018
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+**Combatibility Note: Mobile App v1.13+ is required for Mattermost Server v5.4+**
+
+### Bug Fixes
+- Fixed an issue where the Android app did not allow establishing a network connection with any server that used a self-signed certificate that had the CA certificate user installed on the device.
+- Removed "Copy Post" option on long-press message menu for posts with message menus/dropdowns.
```
Comment (enahum) [inline CHANGELOG.md]: posts without text, it does not matter what else the post contains

**Pull request merged**